### DISCUSSION TITLE
Is OrderedDiGraph deprecated in version 3?

### DISCUSSION THREAD

**Discussion by arossert** [networkx/networkx#6349] [Q&A] [answered]
I want to migrate to the latest release and I noticed that `OrderedDiGraph` no longer exists, I could not find anything in the [Migration guide](https://networkx.org/documentation/latest/release/migration_guide_from_2.x_to_3.0.html) or in the release notes.

What is the replacement for this class? 

Update:
I managed to find [this](https://github.com/networkx/networkx/issues/4614)
Does this mean that I just need to use `DiGraph` as it is already ordered?

**Comment by dschult** [comment, marked as answer]
Because Python dicts became "ordered" in Python 3.6 the NetworkX graph classes also became ordered (in the same way that OrderedGraph is ordered). So, the replacement for `OrderedGraph` is simply `Graph`.  

This change occurred in PR #5813 and it should be mentioned in the release notes and/or migration guide, but it isn't. We'll work to fix that.

**Comment by rossbar** [comment]
> it should be mentioned in the release notes and/or migration guide, but it isn't.

FWIW this is mentioned in the NetworkX 3.0 release, though not very prominently. If you search the [Merged PRs section of the release notes](https://networkx.org/documentation/stable/release/release_3.0.html#merged-prs) you can find a reference to #5813 and the removal of the ordered graph classes. Adding a blurb to the migration guide would certainly be an improvement though!
